docs/en/developer-portal-page.md

```diff
@@ -222,7 +222,7 @@ The following code snippet is an example response for a successful payment:
 
 | PSU-ID            | Iban                   | SCA Status                  | Transaction Status                  |
 | :---------------- | :--------------------- | :-------------------------- | :---------------------------------- |
-| PSU-Successful    | DE11760365688833114935 | finalised                   | ACTC/ACSC\*                         |
+| PSU-Successful    | DE11760365688833114935 | finalised                   | ACTC/ACSP\*                         |
```

Comment:
Future dated and periodic payments also get ACSP as status and not ACTC. Should be ACTC

Reply:
done